samples/scaleout/senderside-and-distributor/Core_6/Shared/OrderValidated.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
+public class OrderValidated :
+    IMessage
```

Comment:
Because the message assembly is shared between V5 and V6. IMessage is used in the custom convention to satisfy both 5 and 6.